src/library/mesh_to_grid_map_converter.cpp

```diff
@@ -95,10 +77,11 @@ bool MeshToGridMapConverter::meshToGridMap(
 
   // Creating the converter
   grid_map::GridMapPclConverter grid_map_pcl_converter;
```

Comment:
Are you actually calling any member function with this object? If it's just a static class, I think the style guide says we should replace it with a namespace and set of simple functions.

Reply:
I removed the unused variables since I'm directly using static methods from their corresponding classes. With these static methods I'm prefixing the namespace where they belong to. Is that okay?

Reply:
That's good. Thanks!